spec/firebase_app_distribution_action_spec.rb

```diff
@@ -27,6 +27,87 @@
     Faraday.default_connection = nil
   end
 
+  describe '#get_value_from_value_or_file' do
+    it 'should return the release notes string' do
+      expect(Fastlane::Actions::FirebaseAppDistributionAction.get_value_from_value_or_file("Hello World", "")).to eq("Hello World")
```

Comment:
Try to make these lines shorter. For example, use an intermediate `result` variable, or maybe save off `Fastlane::Actions::FirebaseAppDistributionAction` into an `action` variable.

Reply:
We have decided to go with the second option since it adds less lines of code and makes a lot of the lines shorter. We did not change all of the names to action since I believe we will have a PR for the refactoring and so these names may all change.